### PR TITLE
uniform log format

### PR DESCRIPTION
**Pull request by renyunkang**
## Description

**What type of PR is this ?:**

1. Uniform log format.
2. The master branch should use the docker image with the tag master to ensure real-time updates. When the new version is released, manually update the corresponding manifest file with the tag release version number.
3. Cleaned some unused code.

### CODE REVIEW

File: pkg/speaker/bgp/bgp/conf.go [hunk 1]
```diff
@@ -28,7 +29,7 @@ func (b *Bgp) HandleBgpGlobalConfig(global *bgpapi.BgpConf, rack string, delete
 	}
 	err = b.updatePolicy(cm)
 	if err != nil {
```
Comment (chaunceyjiang) [inline pkg/speaker/bgp/bgp/conf.go:31]: Is this error not required to be returned?

Comment (renyunkang) [reply pkg/speaker/bgp/bgp/conf.go:31]: > Is this error not required to be returned?

We should return this error.

**Comment by renyunkang** [comment]
kindly ping @chaunceyjiang

**Pull request merged**